gr-dtv/lib/atsc/atsc_trellis_encoder_impl.cc

```diff
@@ -17,7 +17,7 @@
 namespace gr {
 namespace dtv {
 
-static const int DIBITS_PER_BYTE = 4;
+static constexpr int DIBITS_PER_BYTE = 4;
```

Comment:
is this change required for the rest of the changes to work?

Reply:
If not, I'd prefer to keep it separate ... in a different commit at least, if not just not in this PR.

Reply:
My interpretation of the C++ standard is that because `static_assert` [takes a bool constexpr](https://en.cppreference.com/w/cpp/language/static_assert) it needs to be constexpr.

GCC seems to allow just const, but I think it's safer to not assume all compilers behave the same.

My interpretation may be wrong, though.

Anyway, I've split this into its own commit now.

Reply:
Thx!